ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,199 @@
+    }
+
+    private func setupLayout() {
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(widthAnchor.constraint(equalToConstant: Constants.contactWidth))
+        constraints.append(contentsOf: avatarLayoutConstraints())
+
+        avatarView.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(avatarView)
+
+        if let subtitleLabel = subtitleLabel {
+            constraints.append(contentsOf: titleLabelLayoutConstraints())
+            constraints.append(contentsOf: subtitleLabelLayoutConstraints())
+            subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
+            labelContainer.addSubview(subtitleLabel)
+        } else {
+            constraints.append(contentsOf: identifierLayoutConstraints())
+        }
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(titleLabel)
+        
+        labelContainer.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(labelContainer)
+
+        constraints.append(contentsOf: labelContainerLayoutConstraints())
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    private func avatarLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            avatarView.heightAnchor.constraint(equalToConstant: Constants.avatarViewHeight),
+            avatarView.widthAnchor.constraint(equalToConstant: Constants.avatarViewWidth),
+            avatarView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            avatarView.topAnchor.constraint(equalTo: topAnchor),
+            avatarView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            avatarView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ]
+    }
+
+    private func labelContainerLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            labelContainer.widthAnchor.constraint(equalToConstant: Constants.contactWidth),
+            labelContainer.centerXAnchor.constraint(equalTo: centerXAnchor),
+            labelContainer.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel),
+            labelContainer.leadingAnchor.constraint(equalTo: leadingAnchor),
+            labelContainer.trailingAnchor.constraint(equalTo: trailingAnchor),
+            labelContainer.bottomAnchor.constraint(equalTo: bottomAnchor),
+            labelContainer.heightAnchor.constraint(greaterThanOrEqualToConstant: 2 * Constants.labelMinimumHeight),
+            labelContainer.heightAnchor.constraint(lessThanOrEqualToConstant: Constants.titleLabelMaximumHeight + Constants.subtitleMaximumHeight)
+        ]
+    }
+
+    private func titleLabelLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            titleLabel.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel),
```

Comment:
equalTo: labelContainer.topAnchor instead.

A subview should always position itself relative to its superview or other views inside its superview.
Currently, this is a violation of view hierarchy encapsulation.